src/data/services/EnterpriseAccessApiService.ts

```diff
@@ -262,6 +265,27 @@ class EnterpriseAccessApiService {
     const subsidyHydratedGroupLearnersEndpoint = `${EnterpriseAccessApiService.baseUrl}/subsidy-access-policies/${subsidyAccessPolicyUUID}/group-members?${queryParams.toString()}`;
     return EnterpriseAccessApiService.apiClient().get(subsidyHydratedGroupLearnersEndpoint);
   }
+
+  /**
+   * Aggregates subscriptions, course enrollments, and flex group memberships to support the learner profile view
+   * @param {String} userEmail The email address for a learner within an enterprise
+   * @param {String} lmsUserId The unique ID of an LMS user
+   * @param {String} enterpriseUuid The UUID of an enterprise customer
+   * @returns {Promise} - A promise that resolves to the learner profile aggregate response.
+   */
+  static fetchAdminLearnerProfileData(
+    userEmail: string,
+    lmsUserId: string,
+    enterpriseUuid: string,
+  ) : LearnerProfileResponse {
+    const queryParams = new URLSearchParams({
+      user_email: userEmail,
+      lms_user_id: lmsUserId,
+      enterprise_customer_uuid: enterpriseUuid,
+    });
+    const url = `${EnterpriseAccessApiService.baseUrl}/admin-view/learner_profile/?${queryParams.toString()}`;
+    return EnterpriseAccessApiService.apiClient().get(url);
```

Comment:
nit: Can we convert the api payload to camelCase here to be more inline with javascript conventions?

Reply:
None of the other responses convert to camelcase before returning the result, and I feel like the api should just return the base response without changing it so that whoever consumes the result can do what they need to it.

Reply:
While it's true that the other methods in `EnterpriseAccessApiService` don't have the camelCase conversion, it is a pattern we're trying to move toward bit by bit in [LmsApiService](https://github.com/openedx/frontend-app-admin-portal/blob/master/src/data/services/LmsApiService.ts#L497) as we flesh out the TypeScript there.